server/src/test/java/io/crate/analyze/MetadataToASTNodeResolverTest.java

```diff
@@ -527,4 +527,23 @@ public void test_generated_expression_on_geo_shape_in_show_create_table_output()
             Matchers.containsString("\"g\" GEO_SHAPE GENERATED ALWAYS AS 'POLYGON (( 5 5, 30 5, 30 30, 5 30, 5 5 ))")
         );
     }
+    @Test
+    public void test_geo_shape_configurations_shown_for_geo_shape_array() throws Exception {
```

Comment:
This test indeed covers both changes. Was going to remove a new test in DocIndexMetadataTest case but then thought it could be useful to assert internal state of the Reference in case we have 2 bugs in persist/retrieve that cancel each other.

Let's imagine that we by accident persist string into distanceErrorPct and then SHOW CREATE shows Double because of another unexpected casting. Maybe be overthinking but this test is cheap anyway.

Reply:
Yes, I suggested to create a single test that asserts both, the reference representation and the show output

Reply:
removed one test and added ref assertion to another